tests/e2e/default/more-options-menu/offline-user.wdio-spec.js

```diff
@@ -0,0 +1,196 @@
+      await commonPage.openMoreOptionsMenu();
+      expect(await commonPage.isMenuOptionVisible('export', 'contacts')).to.be.false;
+      expect(await commonPage.isMenuOptionVisible('edit', 'contacts')).to.be.false;
+      expect(await commonPage.isMenuOptionEnabled('delete', 'contacts')).to.be.false;
+    });
+
+    it('Report tab - options enabled when sms report selected', async () => {
+      await commonPage.goToReports();
+      expect(await (await commonPage.moreOptionsMenu()).isExisting()).to.be.false;    
+      await reportPage.goToReportById(smsReportId);
+      await commonPage.openMoreOptionsMenu();
+      expect(await commonPage.isMenuOptionVisible('export', 'reports')).to.be.false;
+      expect(await commonPage.isMenuOptionVisible('edit', 'reports')).to.be.false;
+      expect(await commonPage.isMenuOptionEnabled('delete', 'reports')).to.be.true;     
+    });
+
+    it('Report tab - options enabled when xml report selected', async () => {  
+      await reportPage.goToReportById(xmlReportId);
+      await commonPage.openMoreOptionsMenu();
+      expect(await commonPage.isMenuOptionVisible('export', 'reports')).to.be.false;
+      expect(await commonPage.isMenuOptionEnabled('edit', 'reports')).to.be.true;
+      expect(await commonPage.isMenuOptionEnabled('delete', 'reports')).to.be.true;     
+    });
+  });
+
+  describe('all permissions disabled', async () => {
+
+    before(async () => {
+      const allPermissions = ['can_edit', 'can_delete_contacts', 'can_export_all', 
+        'can_export_contacts', 'can_export_messages', 
+        'can_delete_reports', 'can_update_reports'];
+      await utils.updatePermissions(offlineUser.roles, [], allPermissions);
+      await commonPage.closeReloadModal();
+    });
+
+    after(async () => await utils.revertSettings(true));
+  
+    it(' - all tabs, kebab menu not available', async () => {
+      await commonPage.goToMessages();
+      await sms.sendSms('testing', contact.phone);
+      expect(await (await commonPage.moreOptionsMenu()).isExisting()).to.be.false;
+
+      await commonPage.goToPeople();
+      expect(await (await commonPage.moreOptionsMenu()).isExisting()).to.be.false;
+        
+      await contactPage.selectLHSRowByText(contact.name);
+      expect(await (await commonPage.moreOptionsMenu()).isExisting()).to.be.false;  
+    
+      await commonPage.goToReports();
+      expect(await (await commonPage.moreOptionsMenu()).isExisting()).to.be.false;    
+      await reportPage.goToReportById(smsReportId);
+      expect(await (await commonPage.moreOptionsMenu()).isExisting()).to.be.false;
+      await reportPage.goToReportById(xmlReportId);
+      expect(await (await commonPage.moreOptionsMenu()).isExisting()).to.be.false;   
+    });    
+  });
+
+  describe('- DELETE permissions disabled', async () => {
+    before(async () => {
+      await utils.updatePermissions(offlineUser.roles, [], ['can_delete_contacts', 'can_delete_reports']);
```

Comment:
I'm confused with the tests inside this `describe('- DELETE permissions disabled',...`

If the user has `can_edit` and `can_update_reports`, but not `can_delete_contacts` and `can_delete_reports`. So why the menu isn't displayed, it should have the `edit` option, right?

Am I forgetting something? 🤔